vmdb/spec/support/api_spec_helper.rb

```diff
@@ -91,22 +91,22 @@ def init_api_spec_env
       :entrypoint => "/api"
     }
 
-    collections  = %w(auth vms tags providers hosts data_stores resource_pools clusters
-                      policies policy_profiles templates policy_actions events conditions)
+    collections  = %w(auth automation_requests availability_zones clusters conditions data_stores
+                      events flavors groups hosts policies policy_actions policy_profiles providers
+                      provision_requests request_tasks requests resource_pools roles security_groups
+                      servers service_catalogs service_requests service_templates services tags
+                      tasks templates users vms zones)
 
     collections.each { |collection| @cfme["#{collection}_url".to_sym] = "#{@cfme[:entrypoint]}/#{collection}" }
 
     define_user
-    define_url_methods
+    define_url_methods(collections)
```

Comment:
good point, cfme for URL's is no longer needed now that we have the _url methods, will keep around for the other stuff. I'll update this PR for this.  Thanks.